src/jit-dasm/jit-dasm.cs

```diff
@@ -518,41 +512,36 @@ public void GenerateAsm()
                         var assemblyFileName = Path.ChangeExtension(assembly.Name, ".dasm");
                         dasmPath = Path.Combine(_rootPath, assembly.OutputPath, assemblyFileName);
 
-                        PathUtility.EnsureParentDirectoryExists(dasmPath);
+                        Utility.EnsureParentDirectoryExists(dasmPath);
 
                         AddEnvironmentVariable("COMPlus_JitStdOutFile", dasmPath);
                     }
 
-                    Command generateCmd = null;
-
-                    try
-                    {
-                        generateCmd = CreateCommand(new ScriptResolverPolicyWrapper(), commandArgs);
-                    }
-                    catch (CommandUnknownException e)
-                    {
-                        Console.Error.WriteLine("\nError: {0} command not found!\n", e);
-                        Environment.Exit(-1);
-                    }
-
                     if (this.verbose)
                     {
                         Console.WriteLine("Running: {0} {1}", _executablePath, String.Join(" ", commandArgs));
                     }
 
-                    CommandResult result;
+                    ProcessResult result;
 
                     if (_rootPath != null)
                     {
                         var logPath = Path.ChangeExtension(dasmPath, ".log");
+                        result = ExecuteProcess(commandArgs);
 
-                        // Redirect stdout/stderr to log file and run command.
-                        using (var outputStreamWriter = File.CreateText(logPath))
+                        // Write stdout/stderr to log file.
+                        StringBuilder output = new StringBuilder();
+                        if (!string.IsNullOrEmpty(result.StdOut))
+                        {
+                            output.AppendLine(result.StdOut);
```

Comment:
Despite the `AppendLine` this is potentially mutli-line output?

Hoping this doesn't lose or encode line breaks like we see in CI results sometimes.

Reply:
Which CI job are you referring? By the way, thanks for pointing this. It had bug. It should have passed `capture=true` to `ExecuteProcess()` so we get back `result.StdOut`. Also, I have changed to `output.Append()` without adding extra line.

Reply:
One of those where we see lots of long lines with embedded `\r\n` for the captured output, eg:

![image](https://user-images.githubusercontent.com/10121823/93655905-59ebc880-f9db-11ea-8a5f-d0ed2ae0cd6c.png)

Reply:
I see. Sure, I will check it out periodically.